toolchain/codegen/codegen.h

```diff
@@ -0,0 +1,20 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_CODEGEN_CODEGEN_H_
+#define CARBON_TOOLCHAIN_CODEGEN_CODEGEN_H_
+
+#include <cstdint>
+
+#include "llvm/IR/Module.h"
+
+namespace Carbon {
+// Prints the assembly to stdout for the given llvm module.
```

Comment:
Idiomatically, I expect functions to return `true` on success and `false` on failure. Right now this does the reverse. Please make the behavior match the idiom, and document the return behavior for clarity.

Reply:
Although you changed the behavior, please also document the return behavior here.